src/deadline/mcp/config.py

```diff
@@ -0,0 +1,70 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""Configuration definitions for MCP tools in Deadline Cloud."""
+
+from typing import Any, Callable, List, Optional, TypedDict, Dict
+
+from ..client import api
+from .tools import job
+
+
+class ToolConfig(TypedDict):
+    """Configuration for a single MCP tool."""
+
+    func: Callable[..., Any]
+    params: Optional[List[str]]
```

Comment:
Looks like this is being used for lists of parameter names (as opposed to values or definitions). Maybe `param_names` is a better variable name here.